demo/admin/src/pages/Page.tsx

```diff
@@ -54,4 +54,5 @@ export const Page: DocumentInterface<Pick<GQLPage, "content" | "seo">, GQLPageIn
     },
     menuIcon: File,
     hideInMenuIcon: FileNotMenu,
+    getAnchors: (input) => PageContentBlock.getAnchors?.(PageContentBlock.input2State(input.content)) ?? [],
```

Comment:
should be part of DocumentInterface, and should be required (in next)

Reply:
It is part of DocumentInterface, but optional.

> and should be required (in next)

Should we target next, or should it be made required in next?

Reply:
it should be required in next.

if next is possible I would target that, makes things less work

Reply:
I've talked to PM and it is okay to target next, changed the target and made the method required